LEGO1/lego/legoomni/src/build/legocarbuild.cpp

```diff
@@ -771,14 +793,214 @@ MxS32 LegoCarBuild::FUN_10024850(MxLong p_x, MxLong p_y)
 	return result;
 }
 
-// STUB: LEGO1 0x10024890
-// STUB: BETA10 0x1006d512
-undefined4 LegoCarBuild::FUN_10024890(LegoEventNotificationParam* p_param)
+#ifdef NDEBUG
+
+// FUNCTION: LEGO1 0x10024890
+undefined4 LegoCarBuild::FUN_10024890(MxParam* p_param)
```

Comment:
maybe this is MxNotificationParam instead of MxParam?

Reply:
Possible, but the beta strongly suggests that there is a typecast right at the beginning of the function, so the type here should not matter at all.